### ISSUE TITLE
"Ghost" source file in generated sourcemap

### ISSUE THREAD

**Issue by mircolino** [terinjokes/gulp-uglify#109]
```
var gulp = require('gulp');
var tsc  = require('typescript');
var concat = require('gulp-concat');
var minify = require('gulp-uglify');
var sourcemaps = require('gulp-sourcemaps');
var ts = require('gulp-typescript');

var prj = {
  src: '/dev/prj/src/main/webapp/',
  dst: '/dev/prj/build/webapp/main/',

  min: 'src/bundle.min.js',

  tsFiles: [
    'src/config.ts',
    'src/log.ts',
    'src/main.ts'
  ],

  tsProject: null,
  tsConfig: {
    typescript: tsc,
    sortOutput: true,
    module: 'commonjs',
    target: 'ES5',
    declarationFiles: false,
    noExternalResolve: false,
    noImplicitAny: true
  }
};

prj.tsProject = ts.createProject(prj.tsConfig);

gulp.task('transpile', function() {
  gulp.src(prj.tsFiles, {cwd: prj.src, base: prj.src})
    .pipe(sourcemaps.init())
    .pipe(ts(prj.tsProject))
    .pipe(concat(prj.min))
    .pipe(minify())
    .pipe(sourcemaps.write('.'))
    .pipe(gulp.dest(prj.dst));
});
```

When executing the above gulpfile the generated 'bundle.min.js.map' sourcemap contains the following:

```
"sources":["src/config.ts","src/bundle.min.js","src/log.ts","src/main.ts"]
"file":"src/bundle.min.js"
```

'bundle.min.js' is in the list of sources but its source code is not inlined in the sourcemap, resulting in a ghost 'bundle.min.js' source file when debugging in Chrome JavaScript console.

After manually removing "src/bundle.min.js" from the "sources" array in the sourcemap, Chrome JavaScript console works as expected.

How can I tell gulp-uglify not to add "src/bundle.min.js" to the sourcemap?

**Comment by terinjokes** [comment]
@mircolino do you still see this behavior with version 1.4.2?

**Comment by terinjokes** [comment]
Resolved in v1.5.0.
